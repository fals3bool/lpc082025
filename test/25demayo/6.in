3
Carne 0.40
Fideos 0.10
Tomate 0.20
3 3
Carne 0.50
Fideos 0.50
Tomate 0.30
