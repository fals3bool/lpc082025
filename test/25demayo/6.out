Carne 0.7
Tomate 0.3
